jaxrs/src/main/java/org/killbill/billing/jaxrs/json/CatalogJson.java

```diff
@@ -66,8 +65,7 @@ public class CatalogJson {
     private final List<PriceListJson> priceLists;
 
     @JsonCreator
-    public CatalogJson(@JsonProperty("name") final String name,
-                       @JsonProperty("effectiveDate") final Date effectiveDate,
+    public CatalogJson(@JsonProperty("name") final String name, @JsonProperty("effectiveDate") final Date effectiveDate,
```

Comment:
Nit: Can we keep one argument per line - the way it was. In general we chose to break one per line or have all on he same line, not a mix of the two.